packages/web-components/components/document/src/DocumentCaptureScreens.js

```diff
@@ -194,6 +253,14 @@ class DocumentCaptureScreens extends HTMLElement {
     this.documentInstructionBack.addEventListener(
       'document-capture-instructions.upload',
       async (event) => {
+        console.log('dispatch: metadata.document-back-origin', {
+          detail: { imageOrigin: 'uploaded' },
+        });
+        smartCameraWeb.dispatchEvent(
+          new CustomEvent('metadata.document-back-origin', {
+            detail: { imageOrigin: 'uploaded' },
```

Comment:
can we confirm if this is supposed to be `upload` or `uploaded`

Reply:
You were right to doubt that. Ope provided new values for them. There was an error in the story.